go/apps/api/routes/v2_identities_create_identity/handler.go

```
@@ -0,0 +1,222 @@
+			},
+		}
+
+		if req.Ratelimits != nil {
+			for _, ratelimit := range *req.Ratelimits {
+				ratelimitID := uid.New(uid.RatelimitPrefix)
+				err = db.Query.InsertIdentityRatelimit(ctx, tx, db.InsertIdentityRatelimitParams{
+					ID:          ratelimitID,
+					WorkspaceID: auth.AuthorizedWorkspaceID,
+					IdentityID:  sql.NullString{String: identityID, Valid: true},
+					Name:        ratelimit.Name,
+					Limit:       int32(ratelimit.Limit),
```

golangci-lint (1.64.8)

[high] G115: integer overflow conversion int -> int32

(gosec)